indexbuilder.go

```diff
@@ -195,6 +203,9 @@ func NewIndexBuilder(r *Repository) (*IndexBuilder, error) {
 	b := &IndexBuilder{
 		contentPostings: newPostingsBuilder(),
 		namePostings:    newPostingsBuilder(),
+		fileEndSymbol:   []uint32{0},
```

Comment:
why does this start with one entry?

Reply:
`fileEndSymbol[0] == 0` since its the index of the first symbol in the first document.